app/src/main/java/com/infomaniak/mail/ApplicationMain.kt

```diff
@@ -53,12 +53,45 @@ class ApplicationMain : Application() {
 
     override fun onCreate() {
         super.onCreate()
+        if (BuildConfig.DEBUG) configureDebugMode() else configureReleaseMode()
```

Comment:
Currently we don't have a release mode config, so it's useless, just add it if it's necessary